src/components/ResultsEmpty.tsx

```diff
@@ -44,7 +44,7 @@ export const ResultsEmpty: React.FC<ResultsEmptyProp> = ({
         )}
         <Text color='text.subtle' letterSpacing='0.012em' translation={body} />
         {typeof title === 'object' && <Box mt={4}>{title}</Box>}
-        {ctaHref && ctaText && (
+        {ctaText && (
```

Comment:
We don't necessarily need an href, a button may be for an internal action, not necessarily pushing any href in history